### DISCUSSION TITLE
Interrupted Uploads?

### DISCUSSION THREAD

**Discussion by Clue-ess-coder** [vzhd1701/enex2notion#61] [Q&A] [answered]
First of all, ABSOLUTELY PHENOMENAL TOOL

Secondly, is it possible to continue interrupted uploads, or do I have to restart the upload of that notebook?

**Comment by mtalcott** [comment, marked as answer]
Looks like there is an option for that [here](https://github.com/vzhd1701/enex2notion/blob/master/README.md?plain=1#L98):

```
  --done-file FILE           file for uploaded notes hashes to resume interrupted upload
```
